FBSnapshotTestCase/UIImage+Compare.m

```diff
@@ -76,15 +84,53 @@ - (BOOL)compareWithImage:(UIImage *)image
     free(imagePixels);
     return NO;
   }
-    
+
+    /*
+  CGFloat scaleFactor = [[UIScreen mainScreen] scale];
+  CGContextScaleCTM(referenceImageContext, scaleFactor, scaleFactor);
+  CGContextScaleCTM(imageContext, scaleFactor, scaleFactor);
+    */
```

Comment:
I thought it might be that so I tested it by setting the very first pixel in my test be the one that should be different. I got white for both images when scaled to 2x, white vs. black when scaled to 1x (by testing on the iPhone vs. iPad)